config/development.json

```diff
@@ -20,8 +20,8 @@
 	"login_url": "https://wordpress.com/wp-login.php",
 	"logout_url": "https://wordpress.com/wp-login.php?action=logout&redirect_to=https%3A%2F%2F|subdomain|wordpress.com",
 	"facebook_api_key": "249643311490",
-	"dsp_stripe_pub_key": "pk_live_51LYYzQF53KN4RFN0oHFYHpzmHGpuUAXZ6ygDAFxM4XkRArRsET0orrofmsynkd9DhFaOGEsDZ3RC4f8mgI0zhEyN000X8i0Mqx",
-	"dsp_widget_js_src": "https://dsp.wp.com/widget.js",
+	"dsp_stripe_pub_key": "pk_test_51LYYzQF53KN4RFN0iNDEzgRCLRC461l8K3m30wCger66JBFAzgU5f6o1qIIKLLGSZUtzKe4wGJFkU2x4Jn9GRATu00StWVfEDc",
```

Comment:
I think you forgot to revert this change. We need to keep the live version of the key and widget URL in here.

Reply:
nice catch.. sorry for that.. it is clearly a mistake

Reply:
fixed